ecs-cli/modules/utils/compose/convert_task_definition_test.go

```diff
@@ -377,24 +391,19 @@ task_definition:
 	assert.NoError(t, err, "Could not close tempfile")
 
 	ecsParamsFileName := tmpfile.Name()
+	ecsParams, err := ReadECSParams(ecsParamsFileName)
+	assert.NoError(t, err, "Could not read ECS Params file")
+
 	taskRoleArn := "arn:aws:iam::123456789012:role/tweedledum"
 
-	taskDefinition, err := convertToTaskDefWithEcsParamsInTest(t, []string{"mysql", "wordpress"}, &config.ServiceConfig{}, taskRoleArn, ecsParamsFileName)
+	taskDefinition, err := convertToTaskDefWithEcsParamsInTest(t, []string{"mysql", "wordpress"}, &config.ServiceConfig{}, taskRoleArn, ecsParams)
 
 	if assert.NoError(t, err) {
 		assert.Equal(t, "host", aws.StringValue(taskDefinition.NetworkMode), "Expected network mode to match")
 		assert.Equal(t, "arn:aws:iam::123456789012:role/tweedledum", aws.StringValue(taskDefinition.TaskRoleArn), "Expected task role arn to match")
 	}
 }
 
-func TestConvertToTaskDefinitionWithECSParamsWithNoSuchFileError(t *testing.T) {
```

Comment:
Just curious, why was this unit test removed?

Reply:
Moved to ecs_params_reader test, since ConvertTaskDefinition now takes a pointer to ECSParams instead of a filename.